benchmarks/common.py

```diff
@@ -918,6 +918,13 @@ def setup_amp(self):
             self.grad_scaler = torch.cuda.amp.GradScaler()
             self.autocast = torch.cuda.amp.autocast
 
+    def init_optimizer(self, device, params):
+        if device == "cuda":
+            # capturable is only supported on cuda at the moment
+            self.optimizer = torch.optim.Adam(params, capturable=True)
+        else:
+            self.optimizer = None
```

Comment:
```suggestion
    def init_optimizer(self, device, params):
        self.optimizer = torch.optim.Adam(params, capturable=True) if device == "cuda" else None
```

Reply:
I actually have to change this condition due to some failures, I probably won't make it one line, since it will get longer now.